net.h

```diff
@@ -82,7 +83,7 @@ namespace con {
       layers[l]->forward();
     }
 
-    cout << "loss: " << outputLayer->l << endl;
+//    cout << "loss: " << outputLayer->l << endl;
```

Comment:
Please keep this line. We need to see the loss decreasing to verify that our program is working.